osu.Game.Rulesets.Osu/Objects/Drawables/Connections/FollowPoint.cs

```diff
@@ -50,6 +50,7 @@ protected override void LoadComplete()
         {
             base.LoadComplete();
 
+            Delay(-TransformStartTime); // So our follow points start their transforms from t=0 (the start of the map), rather from the lead in time (where the clock is right now).

```

Comment:
TransformStartTime is completely automatic. What you're doing here is *removing* it from the picture based on knowledge you have of parent implementation. Look into the property before blindly using it.

Yes it's currently wrong, but this is not the correct solution.